### PR TITLE
Changed support email address

### PR DESCRIPTION
**Pull request by edwoodward**

### CODE REVIEW

File: src/scripts/pages/about/contact/contact-template.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
   <p>
     <strong>Phone:</strong> 713-348-5012<br />
-    <strong>Email:</strong> cnx@cnx.org
+    <strong>Email:</strong> support@openstax.org
```
Comment (pumazi) [inline src/scripts/pages/about/contact/contact-template.html]: In our last QA round, @kajalp mentioned that this should be clickable email address. Can we make it a mailto item?

Comment (reedstrm) [inline src/scripts/pages/about/contact/contact-template.html]: My thoughts are make the minimum possible change that  can't possibly break, then change it correctly (make this a config property, probably) on master. Would making it a mailto: fit the first or the second?

Comment (edwoodward) [inline src/scripts/pages/about/contact/contact-template.html]: Done

File: src/scripts/pages/about/contact/contact-template.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
   <p>Please feel free to e-mail us for more information.</p>
 
   <h3>Technical Support</h3>
-  <p>If you are encountering a technical problem with our site, please send an email to <a href="mailto:techsupport@cnx.org">techsupport@cnx.org</a>. You can also submit a bug report using our bug submission form.</p>
+  <p>If you are encountering a technical problem with our site, please send an email to <a href="mailto:support@openstax.org">support@openstax.org</a>. You can also submit a bug report using our bug submission form.</p>
   <h3>General Questions</h3>
-  <p>If you have questions about the site, would like to schedule author training, or have any other general questions regarding OpenStax CNX, please send an email to cnx@cnx.org.</p>
+  <p>If you have questions about the site, would like to schedule author training, or have any other general questions regarding OpenStax CNX, please send an email to support@openstax.org.</p>
```
Comment (reedstrm) [inline src/scripts/pages/about/contact/contact-template.html]: should this one be clickable too?

**Pull request merged**